src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,14 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+// eslint-disable-next-line max-len
+export const CommentList: React.FC<{ comments: Comment[] }> = ({ comments }) => (
```

Comment:
figure out a way of removing linter disabling
You should avoid disabling linter